samples/multiplatform/build.gradle.kts

```diff
@@ -0,0 +1,67 @@
+import org.gradle.kotlin.dsl.kotlin
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+plugins {
+    base
+}
+
+buildscript {
+    repositories {
+        mavenCentral()
+    }
+
+    dependencies {
+        classpath(kotlin("gradle-plugin", "1.2.31"))
+        classpath(kotlin("native-gradle-plugin", "0.6.2"))
+    }
+}
+
+allprojects {
```

Comment:
Is all that "boilerplate" really needed?
I think it would be good to define everything in their build.gradle files. Instead of the project build.gradle file..

Reply:
better apply it once for all project